test/WormholeL2VoteAggregator.t.sol

```diff
@@ -163,6 +163,32 @@ contract CastVote is L2VoteAggregatorTest {
     l2VoteAggregator.castVote(1, _support);
   }
 
+  // timeToProposalEnd needs to be a uint32 because the L1Block interface has a max value of uint64
+  function testFuzz_RevertWhen_AfterCastWindow(
+    uint96 _amount,
+    uint8 _support,
+    uint256 proposalId,
+    uint32 timeToProposalEnd
+  ) public {
+    vm.assume(_support < 2);
+    vm.assume(_amount != 0);
+    vm.assume(proposalId != 1); // We have a hardcoded proposal that needs to be removed
```

Comment:
Why can't we test the hardcoded one too?

Reply:
This can be removed, and the reason we needed it was because the hardcoded proposal may be active because of the way it was created in the test.

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/42/commits/88ecc634d0aee2651014e3ff62260019ba8601c2